docs/ZFS.md

```diff
@@ -6,6 +6,11 @@ limited to `min(256 MiB + 0.3% of total zpool size, 20% of system RAM)` but at l
 MiB. Keep in mind that this memory will be taken out form the
 applications.
 
+Also, when calculating memory for applications, EVE takes into account the
+required memory for the operation of ARC, and reserves this amount for the
+needs of the functioning of EVE. Thus, the size of the reserved memory for
+the needs of EVE is increased by the value of `256 MiB + 0.3% of total storage`.
```

Comment:
This conflicts with the last paragraph. Different formulas are given for the same value.
I would remove this paragraph altogether, because the previous one already implies that memory is removed from the calculation:

>this memory will be taken out form the applications

Or, If you wanted to make it more clear, I would remove the last sentence of that paragraph and rephrase your addition to make it absolutely clear that you are talking about the same piece of RAM here. For example

>ZFS provides a rich set of functionality but at a cost of extra
resource usage. Currently ARC (Adaptive Replacement Cache) size is
limited to `min(256 MiB + 0.3% of total zpool size, 20% of system RAM)` but at least 384
 MiB. 
>
>This RAM is included in the memory EVE reserves for its own operational needs. Thus would not be available for allocation for applications.